modules/_node-scripts/bundle/css/processCSSFiles.mjs

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import fs from 'fs/promises';
+import {r2} from 'liferay-theme-tasks';
+import path from 'path';
+
+import {BUILD_RESOURCES_PATH, SRC_PATH} from '../../util/constants.mjs';
+import expandGlobs from '../../util/expandGlobs.mjs';
+
+export default async function processCSSFiles() {
+	const cssFiles = await expandGlobs(['**/*.css'], [], {baseDir: SRC_PATH});
```

Comment:
Do we need to ignore some paths? Like build,classes,node_modules, etc?

Reply:
nope, because of `{baseDir: SRC_PATH}`

Reply:
ah, missed that. sweet.